examples/tests/invalid/structs.test.w

```diff
@@ -45,4 +45,18 @@ struct Showtime extends Razzle, Dazzle {}
 //     ^^^^^^^^ struct "Showtime" extends "Dazzle" which introduces a conflicting member "a"
 
 let x = new cloud.BucketProps(1);
-          //^ Cannot instantiate type "BucketProps" because it is not a class
\ No newline at end of file
+          //^ Cannot instantiate type "BucketProps" because it is not a class
+
+let x = (): str => {
+  struct PreflightStruct {
+      // ^ Structs must be declared at the top-level of a program: Struct PreflightStruct is not defined at the top-level
+    name: str;
+  }
+};
+          
+let y = inflight (): str => {
+  struct InflightStruct {
+      // ^ Structs must be declared at the top-level of a program: Struct InflightStruct is not defined at the top-level
+    name: str;
+  }
+};
```

Comment:
x was already declared before which caused an unrelated/distracting error here, so we should rename these

```suggestion
let inlinePreflightStruct = (): str => {
  struct PreflightStruct {
      // ^ Structs must be declared at the top-level of a program: Struct PreflightStruct is not defined at the top-level
    name: str;
  }
};
          
let inlineInflightStruct = inflight (): str => {
  struct InflightStruct {
      // ^ Structs must be declared at the top-level of a program: Struct InflightStruct is not defined at the top-level
    name: str;
  }
};
```